client/src/components/DashboardTable/DashboardTable.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react'
+import CertificationRow from '../CertificationRow/CertificationRow'
+import { Certificates } from '../../../lib/types'
+import './DashboardTable.css'
+
+interface Props {
+  certificates: Certificates
+}
+
+const DashboardTable = ({ certificates }: Props) => {
+  return (
+    <div className='flex flex-col w-[100%] h-[100%] DashboardTable'>
+
+      <div className='DashboardTable_Filters'>
+      <select className='rounded-md p-2 drop-shadow-xl pr-16' defaultValue='' >
+        <option value='' disabled hidden className='font-semibold'>
+          Industry
+        </option>
+        <option value=''>All</option>
+        <option value='IBM'>IBM</option>
+        <option value='Microsoft'>Microsoft</option>
```

Comment:
We don't have filters for Microsoft, Google, etc.

It is All, Industry, IBM and MyTeam